src/main/java/teammates/common/util/Const.java

```diff
@@ -433,7 +433,8 @@ private Const() {
         public static final String FEEDBACK_QUESTION_RUBRIC_ASSIGN_WEIGHTS =
                 "Assign weights to the columns for calculating statistics.";
 
-        public static final String STUDENT_PROFILE_PICTURE = "Upload a profile picture";
+        public static final String STUDENT_PROFILE_PICTURE = "Upload a profile picture. Max size 5MB. "
+                + ".jpg or .png extensions";
```

Comment:
@Labirinto  Is there any special reason for string concatenation of two constant string?

Reply:
@Progyan1997 I guess that is to avoid checkstyle error (Line should have a max of 125 characters)

Reply:
@Progyan1997, yes it was because of checkstyle error. (Good guess @sukanta-27:))

Reply:
``` java
public static final String STUDENT_PROFILE_PICTURE 
                = "Upload a profile picture. Max size 5MB.  .jpg or .png extensions";
```

This would look better though and satisfy checkstyle as well

Reply:
But the `=` should be on the previous line though. Otherwise it will be another checkstyle error.

Reply:
@sukanta-27 haha thanks fir pointing out. I edited the next moment after posting. :P

Reply:
@tshradheya I guess I responded too quick. :laughing: